Waterminder/Waterminder WatchKit Extension/HealthKitSetupAssistant.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  HealthKitSetupAssistant.swift
+//  Waterminder
+//
+//  Created by Tobias Ruano on 8/02/21.
+//  Copyright © 2021 Tobias Ruano. All rights reserved.
+//
+
+import HealthKit
+
+class HealthKitSetupAssistant {
+    
+    public let healthStore = HKHealthStore()
+    
+    public func requestPermissions() {
+        let dataTypesToWrite : Set = [HKObjectType.quantityType(forIdentifier: HKQuantityTypeIdentifier.dietaryWater)!]
+        
+        healthStore.requestAuthorization(toShare: dataTypesToWrite, read: nil, completion: { (success, error) in
+            if success {
+                print("Authorization complete")
+            } else {
+                print("Authorization error: \(String(describing: error?.localizedDescription))")
+            }
+        })
+    }
+    
+    public func addWater(waterAmount: Double?, forDate : Date) {
+        let quantityTypeWater = HKObjectType.quantityType(forIdentifier: HKQuantityTypeIdentifier.dietaryWater)!
```

Comment:
Please do not force unwrap HKObjectType.quantityType. A guard let statement would be enough